VideoAPI/VideoApi.IntegrationTests/Api/InstantMessages.feature

```diff
@@ -23,3 +23,10 @@ Feature: Instant Messages
     Given I have a valid conference with Nonexistent participants save message request
     When I send the request to the endpoint
     Then the response should have the status BadRequest and success status False
+
+Scenario: Get closed conferences with IMs
```

Comment:
You should have more scenarios here to verify the response code and response  body